src/app/socials/page.tsx

```diff
@@ -1,5 +1,11 @@
-const Socials = () => {
-  return <div>socials comps here</div>;
+import Social from "@/components/socials/Social";
```

Comment:
```suggestion
import SocialBanner from "@/components/socials/Social";
```

update the name for clarity; there's going to be more comps called here